styles.css

```diff
@@ -0,0 +1,689 @@
+h1 {
+    font-size: 2.25rem;
+    font-weight: 700;
+}
+
+h2,
+h3 {
+    font-size: 1.5rem;
+}
+
+header p {
+    font-size: 1.75rem;
+}
+
+#contact p {
+    font-size: 1.25rem;
+}
+
+footer p {
+    font-size: 0.75rem;
+}
+
+#contact label {
+    font-weight: 700;
+    font-size: 0.9rem;
+}
+
+#contact h2 {
+    font-size: 2.25rem;
+}
+
+#introduction p,
+#studies p {
+    font-size: 1.25rem;
+    color: var(--standard-text)
+}
+
+#introduction a,
+#introduction a:visited {
+    color: var(--purple);
+    text-decoration: none;
+}
+
+button {
+    font-size: 1rem;
+}
+
+nav a,
+nav a:visited {
+    text-decoration: none;
+    color: #FFF;
+    font-size: 1rem;
+    font-weight: 700;
+}
+
+#studies h2 {
+    font-size: 2.25rem;
+}
+
+#studies a,
+#studies a:visited {
+    color: var(--purple);
+    text-decoration: none;
+    text-transform: uppercase;
+    font-weight: 700;
+    font-size: 1rem;
+}
+
+#services h3 {
+    color: var(--purple);
+}
+
+#services p {
+    color: var(--service-tile);
+    font-size: 1rem;
+}
+
+/* ------------------------------- 3. Layout --------------------------------------- */
+
+/* --------------- 3.1 Reusables ----------------------- */
+.small-inner-container {
+    display: flex;
+    flex-direction: column;
+    justify-content: space-between;
+    width: 500px;
+    padding-left: 15px;
+    padding-right: 15px;
+}
+
+button {
+    border: none;
+    background-color: var(--pink);
+    color: #FFF;
+    text-transform: uppercase;
+    border-radius: 30px;
+    box-shadow: 0 5px 5px rgba(0, 0, 0, 0.2);
+    cursor: pointer;
+}
+
+/* --------------- 3.2 Other --------------------------- */
+
+/* ------------------------------- 4. Areas ---------------------------------------- */
+
+/* --------------- 4.1 Header ------------------------- */
+header {
+    display: flex;
+    flex-direction: column;
+    justify-content: flex-start;
+    align-items: center;
+    text-align: center;
+    padding-bottom: 200px;
+    background: var(--blue);
+    background: linear-gradient(90deg, var(--blue) 0%, var(--purple) 100%);
+    position: relative;
+    z-index: 0;
+}
+
+/*Adds noise texture to the header image*/
+header::before {
+    content: "";
+    position: absolute;
+    top: 0;
+    left: 0;
+    background-image: url("assets/noise-texture.png");
+    background-repeat: repeat;
+    background-size: auto;
+    width: 100%;
+    height: 100%;
+    z-index: -1;
+}
+
+nav {
+    display: flex;
+    flex-direction: row;
+    justify-content: space-evenly;
+    align-items: center;
+    flex-basis: 60px;
+    width: 100%;
+    flex-grow: 0;
+    flex-shrink: 0;
+    color: #FFF;
+    z-index: 1;
+}
+
+nav ul {
+    display: flex;
+    flex-direction: row;
+    justify-content: space-between;
+    flex-basis: 400px;
+    list-style: none;
+}
+
+header .small-inner-container {
+    align-items: center;
+    flex-basis: 300px;
+    margin-top: 100px;
+    color: #FFF;
+}
+
+header button {
```

Comment:
Gezien dit specifiek voor de header-button is, is het beter om de button netjes een class te geven :)